### DISCUSSION TITLE
The MORT application blocks the activation of hotkeys in Windows (MORT version: 1.274V )

### DISCUSSION THREAD

**Discussion by Truandale** [killkimno/MORT#28] [General]
Hi. I found a strange bug. When running MORT in Windows, the hotkeys that are specified in the properties of application shortcuts are blocked. These hotkeys are needed to quickly launch the application using the hotkey. When MORT is running, the hotkeys stop working. When you turn off MORT, the hotkeys start working. At the same time, the basic keyboard shortcuts continue to work (such as ctrl+alt+del) Only user-defined hotkeys do not work when MORT is running.

For example:
![image](https://github.com/killkimno/MORT/assets/35504283/7871e0db-4f46-4ced-99fd-34f1be900700)
![image](https://github.com/killkimno/MORT/assets/35504283/d70394bb-2280-4b54-86ed-15de53d074c0)

**Comment by kmonkeyhead** [comment]
Hello, thank you for the report
As a result, we found that not only mort but also some other non-installed apps were not working.
Perhaps there is something I need to configure separately.
